CleverPush/Source/CleverPushInstance.m

```diff
@@ -254,16 +255,26 @@ - (id)initWithLaunchOptions:(NSDictionary*)launchOptions channelId:(NSString*)ne
     return [self initWithLaunchOptions:launchOptions channelId:newChannelId handleNotificationReceived:NULL handleNotificationOpened:openedCallback handleSubscribed:subscribedCallback autoRegister:autoRegisterParam];
 }
 
-#pragma mark - Common function to initialize SDK and sync data to NSUserDefaults
 - (id)initWithLaunchOptions:(NSDictionary*)launchOptions
                   channelId:(NSString*)newChannelId
  handleNotificationReceived:(CPHandleNotificationReceivedBlock)receivedCallback
    handleNotificationOpened:(CPHandleNotificationOpenedBlock)openedCallback
            handleSubscribed:(CPHandleSubscribedBlock)subscribedCallback
                autoRegister:(BOOL)autoRegisterParam {
+    return [self initWithLaunchOptions:launchOptions channelId:newChannelId handleNotificationReceived:NULL handleNotificationOpened:openedCallback handleSubscribed:subscribedCallback autoRegister:autoRegisterParam initialized:NULL];
+}
+
+#pragma mark - Common function to initialize SDK and sync data to NSUserDefaults
+- (id)initWithLaunchOptions:(NSDictionary*)launchOptions
+                  channelId:(NSString*)newChannelId
+ handleNotificationReceived:(CPHandleNotificationReceivedBlock)receivedCallback
+   handleNotificationOpened:(CPHandleNotificationOpenedBlock)openedCallback
+           handleSubscribed:(CPHandleSubscribedBlock)subscribedCallback
+               autoRegister:(BOOL)autoRegisterParam initialized:(CPInitializedBlock)initializedCallback {
```

Comment:
Please rename `initialized:(CPInitializedBlock)initializedCallback` to `handleInitialized:(CPInitializedBlock)initializedCallback` and put it into a new line so each parameter has 1 line